src/main/java/com/internet/shop/controller/WelcomeController.java

```diff
@@ -45,18 +49,21 @@ public void init(ServletConfig config) throws ServletException {
         Creating two users
          */
         User firstUser =
-                new User("Ashley", "halsey", "YungbludRocks789");
+                new User("Dariia", "DariiaPikul", "Admin007", ADMIN_ROLE);
         User secondUser =
-                new User("Max", "maxim_dao", "BeautyBlogger456");
+                new User("Ashley", "halsey", "YungbludRocks789", USER_ROLE);
+        User thirdUser =
+                new User("Max", "maxim_dao", "BeautyBlogger456", USER_ROLE);
         userService.create(firstUser);
         userService.create(secondUser);
+        userService.create(thirdUser);
         /*
         Creating shopping carts for the new users
          */
-        ShoppingCart firstCart = new ShoppingCart(firstUser.getId());
         ShoppingCart secondCart = new ShoppingCart(secondUser.getId());
-        shoppingCartService.create(firstCart);
+        ShoppingCart thirdCart = new ShoppingCart(thirdUser.getId());
         shoppingCartService.create(secondCart);
+        shoppingCartService.create(thirdCart);
```

Comment:
let's create a separate InjectDataController for this purpose
we should not place business logic in the init() method